content/docs/implementation-notes.md

```diff
@@ -229,40 +228,40 @@ var node = mount(<App />);
 rootEl.appendChild(node);
 ```
 
-This is working but still far from how the reconciler is really implemented. The key missing ingredient is support for updates.
+Ini bekerja tetapi masih jauh dari bagaimana *reconciler* benar-benar dilaksanakan. Bahan utama yang hilang adalah dukungan untuk pembaruan.
 
-### Introducing Internal Instances {#introducing-internal-instances}
+### Memperkenalkan Instance Internal {#introducing-internal-instances}
 
-The key feature of React is that you can re-render everything, and it won't recreate the DOM or reset the state:
+Fitur utama dari React adalah anda dapat *render* ulang semuanya, dan itu tidak akan membuat ulang DOM atau mengatur ulang *state*:
 
 ```js
 ReactDOM.render(<App />, rootEl);
-// Should reuse the existing DOM:
+// Harus menggunakan kembali DOM yang ada:
 ReactDOM.render(<App />, rootEl);
 ```
 
-However, our implementation above only knows how to mount the initial tree. It can't perform updates on it because it doesn't store all the necessary information, such as all the `publicInstance`s, or which DOM `node`s correspond to which components.
+Namun, implementasi kami di atas hanya tahu cara memasang diagram awal. Itu tidak dapat melakukan pembaruan karena itu tidak menyimpan semua informasi yang diperlukan, seperti semua *`publicInstance`s*, atau yang *`node`s* DOM sesuai dengan komponen yang mana.
```

Comment:
```suggestion
Namun, implementasi kami di atas hanya tahu cara memasang diagram awal. Itu tidak dapat melakukan pembaruan karena itu tidak menyimpan semua informasi yang diperlukan, seperti semua `publicInstance`, atau `node` DOM yang sesuai dengan komponen yang mana.
```

- No need to italicise method names inside code blocks.
- Minor grammar fix.

Reply:
OK, thanks